queries.sql

```diff
@@ -147,4 +147,25 @@ INSERT INTO reports (manifest_id, reason)
 VALUES (
     $1,
     $2
-);
\ No newline at end of file
+);
+
+-- name: get-recent-projects
+SELECT 
+    p.id,
+    p.manifest_id,
+    m.guid AS manifest_guid,
+    e.name AS entity_name,
+    e.type AS entity_type,
+    (SELECT COUNT(*) FROM projects WHERE manifest_id = p.manifest_id) AS entity_num_projects,
+    p.name,
+    p.description,
+    p.webpage_url,
+    p.repository_url,
+    p.licenses,
+    p.tags,
+    p.updated_at
+FROM projects p
+JOIN manifests m ON p.manifest_id = m.id
```

Comment:
- manifest status should be `active`
- Sorting should be by `created_at`. `updated_at` can be updated arbitrarily during crawls.
- Perhaps limit 2 projects per manifest/entity? Some manifests have many projects that'll just dominate the recent listings.

Reply:
Updated